conduct.md

```diff
@@ -67,6 +69,7 @@ This section defines what communication is acceptable and unacceptable. It is vi
   * Members shall
     * Use appropriate softwares for electronic communication.
     * Always participate in the discussion.
+    * Listen to and consider ideas presented by others.
```

Comment:
This should work just fine with what my group has for Communication without sounding too redundant.